sample_app/main_rtk.c

```diff
@@ -987,24 +197,17 @@ int main(void)
 
    g_net = NULL;
 
+   memset(&cmdline_arguments, 0, sizeof(cmdline_arguments));
    strcpy(cmdline_arguments.eth_interface, APP_DEFAULT_ETHERNET_INTERFACE);
```

Comment:
Can/should the cmdline arguments be handled by the shell-command (cmd_pnio_run)? That would allow the user to change the arguments without recompiling the application.

Reply:
Great idea! I have now implemented the station name as optional command line arguments. The paths for Linux LED etc are not relevant here.